test/integration/util/util.go

```diff
@@ -535,7 +534,7 @@ func InitTestAPIServer(t *testing.T, nsPrefix string, admission admission.Interf
 
 	oldCloseFn := testCtx.CloseFn
 	testCtx.CloseFn = func() {
-		cancel()
+		tCtx.Cancel("tearing down apiserver")
```

Comment:
Not only is using ktesting simpler because it requires less lines of code, it's also *better*: determining the reason for "context canceled" errors is hard. When using `tCtx.Cancel` together with helper code that supports cancellation reasons (like the current Gomega) we get more informative error logging.